exasol_udf_mock_python/mock_context.py

```diff
@@ -9,49 +9,136 @@
 
 
 class MockContext(UDFContext):
```

Comment:
I think, at some point, we should extract the UDFContext Interface into its own project, such that we can officially use it for type annotations outside of tests. Could you create a issue for that.

Reply:
It will be better if you create the issue and explain there what you want to achieve.

Reply:
will do